stripe_official.php

```diff
@@ -517,6 +518,22 @@ public function install()
                 Db::getInstance()->execute($sql);
             }
 
+            if (Hook::getIdByName('actionStripeOfficialMetadataDefinition') === false) {
+                $name = 'actionStripeOfficialMetadataDefinition';
+                $title = 'Define metadata of Stripe payment intent';
+                $description = 'Metadata is passing during creation and update of Stripe payment intent';
+                $sql = 'INSERT INTO `' . pSQL(_DB_PREFIX_) . 'hook` (`name`, `title`, `description`) VALUES ("' . pSQL($name) . '", "' . pSQL($title) . '", "' . pSQL($description) . '");';
```

Comment:
pSQL on a field or a table name protect nothing !
On DB_PREFIX no injection possible. It’s environment variable. You need to remove pSQL that is a bad usage. PSQL should always be efficient between two quotes.
In case of variable use bqSQL() that escape back tick ` (so bSQL should always between two back tick to be efficient). Don’t remind bqSQL prevent sql injection but can generate an SQL error. That’s why all parameters used on a sql request sould be validated.